lab1/HelloWorld.java

```diff
@@ -1,5 +1,5 @@
 public class HelloWorld {
     public static void main(String[] args) {
-        System.out.println("Helloo world!");
+        System.out.println("Hellooo world!");
```

Comment:
I think this o is too fat. Reduce it. :hehe:

Reply:
no.